### DISCUSSION TITLE
Build for iOS failing with 'IOServiceAuthorize' is unavailable: not available on iOS

### DISCUSSION THREAD

**Discussion by JacobKingDev** [utmapp/UTM#5719] [Development]
As the title says really, I've configured code signing and downloaded the dependencies from GH actions (though they're just in Downloads, the docs provided no info on where they should be placed, so this could well be the issue). When attempting to build the iOS target through either the provided script or Xcode directly, I get the same compilation error as above.

Pored through the docs but can't find any mention of this.

**Comment by osy** [comment]
It should be placed in the same directory as the xcproject

**Reply by 100tomer** [reply]
I get the same error when trying to run it on an iPhone, but for some reason on an iOS simulator I get a different error, no such file as "qemu"
